app/assets/styles/main.css

```diff
@@ -115,7 +115,7 @@ margin: 0 auto;
 }
 
 .nav-bar {
-  background-color: #1F9DBF;
+  background-color: darkgrey;
```

Comment:
Was there a reason you went with the linguistic HTML color instead of the hex here? Just curious. It's probably snobby but I personally prefer hexes (despite my intense delight over how many of the linguistic colors are named).